tools/yaml-mapper/mapping_datadog_helm_to_datadogagent_crd.yaml

```
@@ -92,21 +92,23 @@ agents.localService.overrideName: spec.global.localService.nameOverride
 agents.networkPolicy.create: ""
 agents.nodeSelector: spec.override.nodeAgent.nodeSelector
 agents.podAnnotations: spec.override.nodeAgent.annotations
-agents.podLabels: ""
+agents.podLabels: spec.override.nodeAgent.labels
+agents.podSecurity: ""
 agents.podSecurity.allowedUnsafeSysctls: ""
 agents.podSecurity.apparmor.enabled: ""
 agents.podSecurity.apparmorProfiles: ""
-agents.podSecurity.capabilities: ""
+agents.podSecurity.capabilities: spec.override.nodeAgent.containers.agent.securityContext.capabilities
 agents.podSecurity.defaultApparmor: ""
 agents.podSecurity.podSecurityPolicy.create: ""
-agents.podSecurity.privileged: ""
-agents.podSecurity.seLinuxContext: ""
-agents.podSecurity.seLinuxContext.rule: ""
-agents.podSecurity.seLinuxContext.seLinuxOptions.level: ""
-agents.podSecurity.seLinuxContext.seLinuxOptions.role: ""
-agents.podSecurity.seLinuxContext.seLinuxOptions.type: ""
-agents.podSecurity.seLinuxContext.seLinuxOptions.user: ""
-agents.podSecurity.seccompProfiles: ""
+agents.podSecurity.privileged: spec.override.nodeAgent.containers.agent.securityContext.privileged
+agents.podSecurity.seLinuxContext: spec.override.nodeAgent.containers.agent.securityContext.seLinuxContext
+agents.podSecurity.seLinuxContext.rule: 
+agents.podSecurity.seLinuxContext.seLinuxOptions.level: spec.override.nodeAgent.containers.agent.securityContext.seLinuxContext.level
+agents.podSecurity.seLinuxContext.seLinuxOptions.role: spec.override.nodeAgent.containers.agent.securityContext.seLinuxContext.role
+agents.podSecurity.seLinuxContext.seLinuxOptions.type: spec.override.nodeAgent.containers.agent.securityContext.seLinuxContext.type
+agents.podSecurity.seLinuxContext.seLinuxOptions.user: spec.override.nodeAgent.containers.agent.securityContext.seLinuxContext.user
+agents.podSecurity.seccompProfiles: spec.override.nodeAgent.containers.agent.securityContext.seccompProfile
```

This one, I haven't been too sure about. I think the operator config, `spec.override.nodeAgent.containers.agent.securityContext.seccompProfile`, takes a configMap, but might worth double-checking

---

Oh actually, I don't think this can be mapped. In the helm chart, `.seccompProfiles` takes a list of strings: https://github.com/DataDog/helm-charts/blob/dcee9e6a25a47795c9a44fc342cc0c284560c8ac/charts/datadog/values.yaml#L1598-L1601

And in the operator, `.seccompProfile` accepts an object, of which neither of the keys would map properly with a list of strings: https://github.com/DataDog/datadog-operator/blob/734366f3c0d151bd3a5d45bbc1a0b62f4e2287ff/config/crd/bases/v1/datadoghq.com_datadogagents.yaml#L3795-L3820

---

Let's leave it blank for now

```suggestion
agents.podSecurity.seccompProfiles: ""
```